src/objects/accounts/migrations/0001_initial.py

```diff
@@ -8,7 +8,6 @@
 
 
 class Migration(migrations.Migration):
-
```

Comment:
Please in the future make a separate PR for linting. And feel free to merge it, because it doesn't affect the code
Now it's very hard to review